src/shipit_code_coverage/shipit_code_coverage/codecov.py

```diff
@@ -73,21 +74,43 @@ def get_commit():
             raise Exception('Mercurial commit is not available yet on mozilla/gecko-dev.')
         return ret
 
+    def rewrite_jsvm_lcov(self):
+        files = os.listdir('ccov-artifacts')
+        for fname in files:
+            if 'jsvm' not in fname or not fname.endswith('.zip'):
+                continue
+
+            zip_file_path = 'ccov-artifacts/' + fname
+            out_dir = 'ccov-artifacts/' + fname[:-4]
+
+            zip_file = zipfile.ZipFile(zip_file_path, 'r')
+            zip_file.extractall(out_dir)
+            zip_file.close()
+
+            lcov_files = [os.path.abspath(os.path.join(out_dir, f)) for f in os.listdir(out_dir)]
+            run(['gecko-env', './mach', 'python', 'python/mozbuild/mozbuild/codecoverage/lcov_rewriter.py'] + lcov_files, cwd=self.repo_dir)
```

Comment:
The command fails pretty often because it cannot map all the JS files with the correct paths, we're fixing it upstream, but in the meantime we need to ignore the errors.